driver/driver.go

```diff
@@ -111,6 +111,10 @@ type Database interface {
 	DescribeCollection(ctx context.Context, collection string, options ...*CollectionOptions) (*DescribeCollectionResponse, error)
 
 	Events(ctx context.Context, collection string, options ...*EventsOptions) (EventIterator, error)
+
+	Publish(ctx context.Context, collection string, doc Document, options ...*PublishOptions) (*PublishResponse, error)
```

Comment:
should we use different type here. For document collection the model is document for pubsub collection it is Message wdyt?

Reply:
yeah better to define a different type.